### PR TITLE
added maths section and numberous courses underit, different headings for different topics in free-courses-hi.md

### PR DESCRIPTION
**Pull request by swastikgithub09**
…bability

## What does this PR do?
Add resource(s) 

### Description
added mathematics and two subheadings-discreteMaths and Stats and pro…

### Why is this valuable (or not)?
Math is an important subject for this field so i have added some basic courses in maths for the betterment of students
### How do we know it's really free?
free on youtube 
### For book lists, is it a book? For course lists, is it a course? etc.
courses
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
for 3 courses, we only need one category for Mathematics

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -196,6 +197,13 @@
 * [Machine Learning Tutorials For Beginners Using Python in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ai6fAMHp-acBmJONT7Y4BSG) - CodeWithHarry (YouTube)
 
 
+### Mathematics
+
+* [Discrete Mathematics](https://www.youtube.com/playlist?list=PLxCzCOWd7aiH2wwES9vPWsEL6ipTaUSl3) - Gate Smashers (YouTube)
```
Comment (davorpa) [inline courses/free-courses-hi.md:202]: Add instructors and remove non-needed YT playlist annotations

```suggestion
* [Discrete Mathematics](https://www.youtube.com/playlist?list=PLxCzCOWd7aiH2wwES9vPWsEL6ipTaUSl3) - Gate Smashers, `tch.:` Varun Singla, `tch.:` Naina Wadhwa Singla
```

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -196,6 +197,13 @@
...
+* [Discrete Mathematics (Full Course) By Dr.Gajendra Purohit](https://www.youtube.com/playlist?list=PLU6SqdYcYsfJ27O0dvuMwafS3X8CecqUg) - Gajendra Purohit (YouTube)
```
Comment (davorpa) [inline courses/free-courses-hi.md:203]:
```suggestion
* [Discrete Mathematics (Full Course) By Dr.Gajendra Purohit](https://www.youtube.com/playlist?list=PLU6SqdYcYsfJ27O0dvuMwafS3X8CecqUg) - Gajendra Purohit
```

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -196,6 +197,13 @@
...
+* [Statistics And Probability](https://www.youtube.com/playlist?list=PLU6SqdYcYsfLRq3tu-g_hvkHDcorrtcBK) - Gajendra Purohit (YouTube)
```
Comment (davorpa) [inline courses/free-courses-hi.md:204]:
```suggestion
* [Statistics And Probability](https://www.youtube.com/playlist?list=PLU6SqdYcYsfLRq3tu-g_hvkHDcorrtcBK) - Gajendra Purohit
```

**Review by davorpa — CHANGES_REQUESTED** [review]

**Review by davorpa — APPROVED** [review]

**Pull request merged**